### ISSUE TITLE
Support for switching accounts

### ISSUE THREAD

**Issue by davidben** [GoogleChrome/chromium-dashboard#1163]
**Is your feature request related to a problem? Please describe.**
I accidentally selected my google.com account and now the doesn't seem to let me switch accounts to my chromium.org one.

**Describe the solution you'd like**
In Monorail, there's a "switch accounts" entry in the dropdown. If chromestatus.com could support something similar, that'd be great.

**Additional context**

**Comment by jstenback** [comment]
Yes, this is pretty terrible. The only way to switch accounts is to sign out, which signs you out of your @google.com account all together, then sign in with the right account. This has been discussed before, thanks for filing an issue on it. I believe Jason has a plan for how to approach this already.

**Comment by davidben** [comment]
I tried signing out earlier today and even then it didn't work. :-( Seemed like something (Google Sign-In?) remembered my selection.

Not a dealbreaker since, worst case, I can work around it with incognito. But it is somewhat annoying. :-) Glad to hear folks have been thinking about it!

**Comment by jrobbins** [comment]
The "Switch accounts" command that I implemented for Monorail is strictly for GAE with Py2 because it uses the 'users' library.  ChromeStatus also currently uses that same library, but we are trying to switch to Google Sign-In instead.  I think that we could probably figure out a switch accounts command with Google Sign-In, but I don't want to implement that functionality for 'users' first and then have it block our transition to Sign-In.

So, this should be blocked on #1201.

**Comment by shivamag00** [comment]
UPDATE: #1201 has been closed now as ChromeStatus is now using Google Sign In.
